caluma-demo/app/templates/form-builder.hbs

```diff
@@ -0,0 +1 @@
+{{outlet}}
```

Comment:
This file should not be needed

Reply:
This was generated automatically when I created the route form-builder. Removed it.